sklearn/model_selection/tests/test_search.py

```diff
@@ -1787,3 +1787,30 @@ def get_n_splits(self, *args, **kw):
                              'inconsistent results. Expected \\d+ '
                              'splits, got \\d+'):
         ridge.fit(X[:train_size], y[:train_size])
+
+
+def test_search_cv__pairwise_property():
+    """
+    Test implementation of BaseSearchCV has the _pairwise property
+    which matches the _pairwise property of its estimator.
+    """
+
+    class PairwiseCV(BaseSearchCV):
+        def __init__(self, estimator, **kwargs):
+            super().__init__(estimator, **kwargs)
+
+    # simple estimator with _pairwise property
+    est = BaseEstimator()
+
+    attr_message = "BaseSearchCV _pairwise property must match estimator"
+
+    # check estimator with and without _pairwise
+    for _pairwise_setting in [True, False]:
+        setattr(est, '_pairwise', _pairwise_setting)
+        cv = PairwiseCV(est)
+
+        # check if cv is pairwise
+        cv_is_pairwise = getattr(cv, '_pairwise', False)
```

Comment:
why not just use `cv._pairwise`?

Reply:
I was following the convention of [`safe_split`](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/utils/metaestimators.py#L145) but hit the 80 character limit so I broke it into a separate line. I suppose since checking `cv._pairwise` won't throw an error for not being instantiated, we can just use it here. I'll make the swap